apps/sso/components/login-card/login-card.tsx

```diff
@@ -105,8 +105,8 @@ export function LoginCard() {
 export default LoginCard;
 
 const StyledLoginCard = styled.div`
-  min-width: 55vw;
-  max-width: 90vw;
+  width: 35vw;
```

Comment:
The width being scaled by viewport makes is look really thin on mobile (or anything smaller than desktop actually). I recommend changing this to min-width: 35vw or just keeping it as auto